ApplicationLibCode/Application/RiaPreferences.cpp

```diff
@@ -270,9 +274,13 @@ RiaPreferences::RiaPreferences()
 
     CAF_PDM_InitFieldNoDefault( &m_osduPreferences, "osduPreferences", "osduPreferences" );
     m_osduPreferences = new RiaPreferencesOsdu;
+    CAF_PDM_InitField( &m_deleteOsduToken, "deleteOsduToken", false, "" );
+    caf::PdmUiPushButtonEditor::configureEditorLabelHidden( &m_deleteOsduToken );
 
     CAF_PDM_InitFieldNoDefault( &m_sumoPreferences, "sumoPreferences", "sumoPreferences" );
     m_sumoPreferences = new RiaPreferencesSumo;
+    CAF_PDM_InitField( &m_deleteSumoToken, "deleteSumoToken", false, "" );
+    caf::PdmUiPushButtonEditor::configureEditorLabelHidden( &m_deleteSumoToken );
```

Comment:
Disable IO to avoid storing bool value: m_deleteSumoToken.xmlCapability()->disableIO();